airbyte-integrations/connectors/destination-gcs/build.gradle

```diff
@@ -1,51 +1,17 @@
 plugins {
-    id 'application'
     id 'airbyte-java-connector'
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.2.0'
-    features = ['db-destinations']
-    useLocalCdk = false
+    cdkVersionRequired = '0.17.0'
+    features = ['db-destinations', 's3-destinations', 'gcs-destinations']
```

Comment:
Do you still need `gcs-destinations` here since the module is just `s3-destinations` ?